src/modules/simulator/simulator_config.h.in

```diff
@@ -0,0 +1,39 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2016 Anton Matosov. All rights reserved.
```

Comment:
Emmm... I am not part of that team, so I don't think I should put your name on my changes.